Formula/h/httping.rb

```diff
@@ -5,6 +5,11 @@ class Httping < Formula
   sha256 "37da3c89b917611d2ff81e2f6c9e9de39d160ef0ca2cb6ffec0bebcb9b45ef5d"
   license "GPL-3.0-only"
 
+  livecheck do
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
  livecheck do
```

The `stable` URL is a GitHub tag archive (not a release asset), so we should explain why the `GithubLatest` strategy is used here.